experimenter/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_ready_for_review_serializer.py

```diff
@@ -242,7 +246,13 @@ def test_invalid_experiment_requires_min_version_less_than_max_version(self):
             NimbusExperiment.Application.FOCUS_IOS,
         ]
     )
-    def test_rollout_min_version_under_115_shows_warning(self, application):
+    @mock.patch(
+        "experimenter.features.manifests.nimbus_fml_loader.NimbusFmlLoader.get_fml_errors",
+    )
+    def test_rollout_min_version_under_115_shows_warning(
+        self, application, mock_get_fml_errors
+    ):
+        mock_get_fml_errors.return_value = []
```

Comment:
If you need to mock this out everywhere you can do it at the class level.  You can define a [mixin class](https://github.com/mozilla/experimenter/blob/main/experimenter/experimenter/kinto/tests/mixins.py) that sets up all the mocking and then just [inherit from it in the test class](https://github.com/mozilla/experimenter/blob/main/experimenter/experimenter/kinto/tests/test_client.py#L14) and it'll do all the mocking for all the tests.

Reply:
this is exactly what I was looking for! ty!!